### PR TITLE
Fix minor error-handling bug

### PR DESCRIPTION
**Pull request by critiqjo**

### CODE REVIEW

**Comment by critiqjo** [comment]
I signed it!

File: tensorflow/contrib/seq2seq/python/ops/helper.py [hunk 1]
```diff
@@ -443,7 +443,7 @@ def sample(self, time, outputs, state, name=None):
     # Outputs are logits, use argmax to get the most probable id
     if not isinstance(outputs, ops.Tensor):
       raise TypeError("Expected outputs to be a single Tensor, got: %s" %
```
Comment (caisq) [inline tensorflow/contrib/seq2seq/python/ops/helper.py]: it would be clearer to say "got type: ".

**Review by caisq — CHANGES_REQUESTED** [review]

File: tensorflow/contrib/seq2seq/python/ops/helper.py (hunk 1 above)
Comment (critiqjo) [inline tensorflow/contrib/seq2seq/python/ops/helper.py]: Sure, I was actually borrowing the pattern from [right above it](https://github.com/critiqjo/tensorflow/blob/84f0cce/tensorflow/contrib/seq2seq/python/ops/helper.py#L337-L338). Should I change that too?

Comment (caisq) [inline tensorflow/contrib/seq2seq/python/ops/helper.py]: OK. In that case never mind. On second thought, the alternative I suggested was not 100% clear, either. What you have is fine and shorter.

**Review by caisq — APPROVED** [review]

**Comment by caisq** [comment]
@tensorflow-jenkins test this please

**Pull request merged**